src/vdom.ml

```diff
@@ -202,7 +202,7 @@ let patchVNodesOnElems_PropertiesApply_Add callbacks elem _idx = function
   | Attribute (namespace, k, v) -> Web.Node.setAttributeNsOptional elem namespace k v
   | Data (k, v) -> Js.log ("TODO:  Add Data Unhandled", k, v); failwith "TODO:  Add Data Unhandled"
   | Event (name, handlerType, cache) -> cache := eventHandler_Register callbacks elem name handlerType
-  | Style s -> List.fold_left (fun () (k, v) -> Web.Node.setStyleProperty elem k (Js.Null.return v)) () s
+  | Style s -> Belt.List.reduceU s () (fun [@bs] () (k, v) -> Web.Node.setStyleProperty elem k (Js.Null.return v))
```

Comment:
what is `reduceU`?  Never heard this function name and have no clue what it does with that odd `U` postfix (also, why is it not following OCaml standards of snake_case for standard library calls?).  What does it do and why is it being used?

Reply:
from the docs:
```
 For higher order functions, it will be suffixed U if it takes uncurried callback.

      val forEach  : 'a t -> ('a -> unit) -> unit
      val forEachU : 'a t -> ('a -> unit [@bs]) -> unit
    

In general, uncurried version will be faster, but it may be less familiar to people who have a background in functional programming. 
```
https://bucklescript.github.io/bucklescript/api/Belt.html

Reply:
Shouldn't uncurrying be part of the optimizer and not something the coder has to deal with?  o.O